std/format/package.d

```diff
@@ -792,21 +792,25 @@ private void formatReflectTest(T)(ref T val, string fmt, string formatted, strin
         {
             alias aa1 = val;
             alias aa2 = val2;
-            assert(aa1 == aa2);
+            assert(aa1 == aa2, "Identical associative arrays differ.");
```

Comment:
This error message could be improved: "Associative arrays differ, but should be identical"